editor/editor_export.cpp

```diff
@@ -1803,6 +1803,41 @@ bool EditorExportPlatformPC::can_export(const Ref<EditorExportPreset> &p_preset,
 	valid = dvalid || rvalid;
 	r_missing_templates = !valid;
 
+#ifdef WINDOWS_ENABLED
+	String rcedit_path = EditorSettings::get_singleton()->get("export/windows/rcedit");
+	if (rcedit_path.is_empty()) {
+		err += TTR("The path of rcedit is empty. No icon or app information data will be included.") + "\n";
+	}
+
+	String icon_path = ProjectSettings::get_singleton()->globalize_path(p_preset->get("application/icon"));
+	if (!icon_path.is_empty() && !FileAccess::exists(icon_path)) {
+		err += TTR("Invalid icon path: ") + "\n" + icon_path + "\n";
```

Comment:
```suggestion
		err += TTR("Invalid icon path: ") + icon_path + "\n";
```
Each line gets a `- ` prefix automatically as if it's a new warning, so it's better to keep this one on a single line too.